site2/docs/admin-api-topics.md

```diff
@@ -973,6 +973,90 @@ admin.topics().getBacklogSizeByMessageId(topic, messageId);
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+
+### Configure deduplication snapshot interval
+
+#### Get deduplication snapshot interval
+
+To get the topic-level deduplication snapshot interval, use one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Pulsar-admin API-->
+
+```
+$ pulsar-admin topics get-deduplication-snapshot-interval options
```

Comment:
```suggestion
pulsar-admin topics get-deduplication-snapshot-interval options
```

It's easier for users to copy the command without $. Please check all occurrences in this PR.